object-group-game/Database/UserModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace object_group_game.Database
+{
+	public class Login
+	{
+		public string Username { get; set; }
+		public string Password { get; set; }
+	}
+}
```

Comment:
This class shouldn't exist, we have Player.cs class for this and we won't ever need to save the user's password locally. Insert it the first time into the database, and then only check with the database if the input password matches. This is how it should go.

Reply:
This class describes the table structure in the database and is only really used to query the database to my understanding.